src/Taboo/TurnManager.py

```diff
@@ -127,21 +129,40 @@ def processDiscard(self, qmsg):
         lastWord = self._wordsByTurnId[self._curTurnId][-1]
 
         if qmsg.jmsg[2] != lastWord.wordId:
-            self._txQueue.put_nowait(ClientTxMsg(["DISCARD-BAD",
-                                                  "Discarding invalid word"],
+            self._txQueue.put_nowait(ClientTxMsg([badReplyType,
+                                                  "invalid word"],
```

Comment:
Similarly, "Invalid word" perhaps?

(We might end up showing these in the frontend so consistency would be good)

Reply:
Fixed in e39a7785eb889868aa5504df39efe2c5b681b651 and pushed.